samples/sampler/stories/default/ImageItem.js

```diff
@@ -0,0 +1,27 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {text, select} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import ImageItem, {ImageItemBase} from '@enact/agate/ImageItem';
+
+ImageItem.displayName = 'ImageItem';
+const Config = mergeComponentMetadata('ImageItem', ImageItem, ImageItemBase);
+
+storiesOf('Agate', module)
```

Comment:
Could you add qa-sampler `storiesOf('Agate QA.ImageItem', module)` for the ImageItem without `children` because there is no sample for it?

Reply:
I've added qa-sampler for `ImageItem`. You can find it in _sampler/stories/default/ImageItemKitchenSink.js_.
Please let me know if this is the correct way of adding qa-sampler.

Reply:
Hi @adrian-cocoara-lgp 
I had to explain how to do it in detail.
Your `ImageItemKitchenSink` sampler was pretty much well.
I just moved it to the ImageItem default sampler.
Thanks for creating it.